pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "1.1.1"
+version = "1.1.2"
```

Comment:
Is there a chance that this could be backwards breaking?

Reply:
Doubtful it would break anything, it will at most delete the existing log subscription filters and create them again.

To your other comment, I moved the prefixes to be prefixes because it was touching similar code. You know scouting rule and all that, leave things better than how you found them.